add_catch(power_of_two test.cpp)
